### ISSUE TITLE
Class not registered (0x80040154 (REGDB_E_CLASSNOTREG))

### ISSUE THREAD

**Issue by Giremo** [takkun-mint/FallGuysTools#10]
I decided to try out your app since I'm interested with the features this tool contain, however I'm suddenly met with application defined or object error upon use
![Screenshot 2024-05-29 223344](https://github.com/takkun-mint/FallGuysTools/assets/171162828/194278ee-65fc-4e7e-81c7-488fa1a506ff)

If I do press continue I will get a white cutout while outside of Fall guys

![Screenshot 2024-05-29 223403](https://github.com/takkun-mint/FallGuysTools/assets/171162828/d5079b10-4e63-4e43-bace-872af4a85d53)

However once I start playing the game then the Class not Registered Error happens, fully preventing me from interacting with the app and clicking continue will continously send out the same error 
![Screenshot 2024-05-29 223524](https://github.com/takkun-mint/FallGuysTools/assets/171162828/a7d43174-7f6b-40f8-9dfa-8ed20dd6cee1)

Heres the Text Error regarding Class not registered
[Error.txt](https://github.com/takkun-mint/FallGuysTools/files/15486089/Error.txt)

I use Steam version of Fall guys if that's important

I also download the latest Net framework 4.8.1 including .Net 8.0 in an attempt to fix the problem

I tried going back to previous versions and only found myself able to use the 4.99.561 version. Anything starting the 4.99.6 and above versions gives the same result error result even if the update notification is ignored

**Comment by takkun-mint** [comment]
Thank you for contacting us.
Sorry for the delay in replying.

I have checked the above error and the source code,
We have found that either “Windows Media Player” is not installed on the PC in question or the ActiveX control for Windows Media Player is not applied.
As a tentative measure, we recommend that you perform the above installation, which will most likely improve the situation.

However, since this Media Player-affiliated program is currently not running and is also a legacy feature, we would like to examine whether you plan to use it in the future and whether to remove it.

Thank you for your inquiry.
Thank you for your continued support of Fall Guys Tools.

**Comment by Giremo** [comment]
Hello! As for your recommendations I have installed the Windows Media Player from the Microsoft store. However the issue still remained. 

![Screenshot 2024-06-04 171649](https://github.com/takkun-mint/FallGuysTools/assets/171162828/fc1e767e-e641-4070-b57e-572ec2adab4e)

I'm not sure how I can deal with this Active X control situation so I assumed it must be in the services but ultimately still gotten the same result.


![Screenshot 2024-06-04 172050](https://github.com/takkun-mint/FallGuysTools/assets/171162828/517fa778-9367-4efd-ada2-4616f7ac2113)

If you have anymore suggestion please advise. 
Thank you for your response

**Comment by takkun-mint** [comment]
Sorry,
By Windows Media Player, I meant the legacy version.
Just in case you opened the registry editor,
Check if there is a key {6BF52A52-394A-11D3-B153-00C04F79FAAA6} in “HKEY_CLASSES_ROOT\TypeLib”.
If there is no corresponding key,
Please reinstall Windows Media Player (Legacy) or use a command prompt opened with administrator privileges.
If not, please reinstall Windows Media Player (Legacy) or run “regsvr32 wmp.dll” at a command prompt opened with administrator privileges.

Thank you again for your cooperation.

**Comment by Giremo** [comment]
there's no corresponding key called 6BF in the directory you are trying to lead me into.

I got WMP legacy installed and also attempted running the command but the failed.

Result is still a no go

![Screenshot 2024-06-06 220935](https://github.com/takkun-mint/FallGuysTools/assets/171162828/2022a978-a30b-4426-85d9-3ae382f45605)

**Comment by takkun-mint** [comment]
Hello!
The text file you attached is 0kB.
I could not check the situation...
Aside from that, in Ver 4.99.95 that was just released
I have disabled all the problematic features.
It should now work in the meantime.
If updating Tools by itself is too hard, please try reinstalling it.
Thank you for your inquiry!
Please feel free to contact me again if you have any further questions!

**Comment by Giremo** [comment]
My apologies, Here's the new text error file that I'm getting the most recent attempt

[New Error.txt](https://github.com/user-attachments/files/15737690/New.Error.txt)

I'll update to let you know if the newest update allows me to use it.

Thank you!

**Comment by takkun-mint** [comment]
After all, to check the error content,
I think it is due to the fact that “ActiveX” is not supported.
I am also a layman, so I don't know for sure, but I think it is a problem that occurs in certain environments, since there have been no indications of such a problem until now.
The function you pointed out is a function that used to allow videos to be played using “AxWindowsMediaPlayer” in the part of the program that was reproduced by Tools before the Creative thumbnail function was implemented.
However, that function was only used as a prank among friends, and although we introduced it, we commented out all the other uses because we could not think of any other use for it.

In summary, we have concluded that the problem was caused by some environmental issue that “ActiveX” is not supported, but that the removal of the unused function solved the problem and does not cause any problems with the current operation of Tools.

There is a lot I don't know about this issue, so I will not close this Issue to help solve the problem.

Thank you for your inquiry.
Please keep up the good work with Fall Guys Tools.

**Comment by Giremo** [comment]
So far, I have made someone else, try the old version of your fg tool, it seems like it does work on w11. 

I have decided to also reinstall my windows 11 os, albeit a custom one and it seems to be working just as fine, but I do have to reinstall everything back which is kind of time consuming. Otherwise I did wonder if there's actually any solution to this problem when I'm still on that broken os.

For now I do think this problem is solely mine alone.
